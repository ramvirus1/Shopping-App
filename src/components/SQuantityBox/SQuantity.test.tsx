import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import SQuantity from "./index";

const props = {
    quantity: 0,
    onQuantityUpdate: (quantity: number) => {}
};

describe('SQuantity', () => {
  const snapshot = renderer.create(<SQuantity {...props} />).toJSON();
  it('should render', () => {
    expect(snapshot).toMatchSnapshot();
  });
});