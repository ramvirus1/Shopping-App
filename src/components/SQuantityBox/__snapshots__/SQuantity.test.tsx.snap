// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SQuantity should render 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "#000",
        "borderRadius": 10.714285714285714,
        "borderStyle": "solid",
        "borderWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
      },
    ]
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "marginBottom": 12.857142857142858,
        "marginLeft": 12.857142857142858,
        "marginRight": 12.857142857142858,
        "marginTop": 12.857142857142858,
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
</View>
`;
